src/palakSJun20/ExtractItemIdFromUrl.java

```diff
@@ -0,0 +1,50 @@
+/*Extract ItemId from below input
+
+	String input = https://www.facebook.com/technocredits/temId=12345/
+	String input = https://www.facebook.com/technocredits/empid=123&itemId=12345/
+	String input = https://www.facebook.com/technocredits/itemId=12345&empid=123/
+	String input = https://www.facebook.com/technocredits/empid=123/
+
+	Output : 
+	item id is 12345 
+	        OR
+	no item id present.*/
+package palakSJun20;
+
+public class ExtractItemIdFromUrl {
+	void findIdFromUrl(String input) {
+		String output = "";
+		if (input.contains("itemId")) {
+			for (int index = input.indexOf("itemId"); index < input.length(); index++) {
+				if (input.charAt(index) >= '0' && input.charAt(index) <= '9') {
+					output += input.charAt(index);
+					if (!(input.charAt(index + 1) >= '0' && input.charAt(index) <= '9')) {
+						break;
+					}
+				}
+			}
```

Comment:
Suggesting an alternative You can also write : 
try{
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

}catch(StringIndexOutOfBoundsException siob){
			System.out.println("Item ID -> Not present");
		}

This two line in try block and it will work.